posthog/api/person.py

```diff
@@ -156,6 +156,17 @@ def merge(self, request: request.Request, pk=None, **kwargs) -> response.Respons
         person = Person.objects.get(pk=pk, team_id=self.team_id)
         person.merge_people([p for p in people])
 
+        data = PersonSerializer(person).data
+        for p in people:
+            for distinct_id in p.distinct_ids:
```

Comment:
This logic should be in `PersonSerializer`

Reply:
Hm I'm not sure? It's logic that's only relevant to this function. I guess I could send it as context but not sure what we win with that

Reply:
Hm I'm not sure? It's logic that's only relevant to this function. I guess I could send it as context but not sure what we win with that